validator/keymanager/remote-web3signer/internal/client.go

```diff
@@ -116,26 +119,34 @@ func (client *ApiClient) GetServerStatus(ctx context.Context) (string, error) {
 
 // doRequest is a utility method for requests.
 func (client *ApiClient) doRequest(ctx context.Context, httpMethod, fullPath string, body io.Reader) (*http.Response, error) {
+	ctx, span := trace.StartSpan(ctx, "remote_web3signer.client.doRequest")
+	defer span.End()
+	b, err := io.ReadAll(body)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to read body for request")
+	}
+	span.AddAttributes(
+		trace.StringAttribute("httpMethod", httpMethod),
+		trace.StringAttribute("fullPath", fullPath),
+		trace.StringAttribute("body", string(b)),
```

Comment:
Attaching the body to a tracing request can have performance issues. Tracing data storage may balloon and uploading the data can be difficult. Please consider removing the body from the span.